python/lsst/obs/subaru/strayLight/yStrayLight.py

```diff
@@ -120,6 +115,7 @@ def run(self, sensorRef, exposure):
 
         model *= exposure.getInfo().getVisitInfo().getExposureTime()
         exposure.image.array -= model
+    # CZW: Should these methods be moved into the class?  They're not useful outside it, right?
```

Comment:
Did this get resolved?  If so, what's the answer?

Reply:
I've removed the comment and left the methods independent of the class.  This is probably overly conservative, but working through the straylight correction is a bit out of the scope of the ticket.  I've added a new ticket DM-16805 to handle a future examination and rewrite of the subaru straylight code.